src/core/Akka.Streams.Tests/Util/IteratorAdapterSpec.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
You need to put the legal header on top of the file, you can just copy it from one of the other files and change the file name.